### DISCUSSION TITLE
Feature Request - Option to add wall(s) around snug supports

### DISCUSSION THREAD

**Discussion by djngnear** [OrcaSlicer/OrcaSlicer#619] [Ideas]
I'd really like the option to add walls or a sheath around snug support structures. It not only adds strength and stability but also gives the interface lines a better foundation. A lot of times I'll find the interface just extruding onto nothing so I'll have to try to mess with more settings to get it right. Please take this into consideration. Thanks.

**Comment by djngnear** [comment]
Anyway you could look into this like BambuStudio already did??

**Comment by lanpartyceo** [comment]
2nd on this, PrusaSlicer has it as functional as such:

![image](https://github.com/user-attachments/assets/56ec4871-a25b-4816-9f1d-cb02537b1fce)

It would apply to grid and snug supports.

Edit: Appears to be a duplicate request of https://github.com/SoftFever/OrcaSlicer/issues/3825
